rts/lra/narayana-lra/src/main/java/io/narayana/lra/filter/ServerLRAFilter.java

```diff
@@ -464,8 +463,9 @@ private void resumeTransaction(URI lraId) {
         // nothing to do
     }
 
-    private String getCompensatorId(URI lraId, URI baseUri, String path) {
-        Map<String, String> terminateURIs = NarayanaLRAClient.getTerminationUris(resourceInfo.getResourceClass(), baseUri, path);
+    private String getCompensatorId(URI lraId, URI baseUri, String path, Long timeout) {
```

Comment:
So this is the only functional change made in the PR.

Reply:
I don't understand what this comment should mean.

Reply:
Sorry, it was just an observation and incorrect. The other functional change is to `getTerminatorURIs` where you had to delete the code that read the participant timeout annotation. Please ignore.